src/Symfony/Component/Validator/Tests/Constraints/WordCountTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Constraints\WordCount;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+use Symfony\Component\Validator\Exception\MissingOptionsException;
+use Symfony\Component\Validator\Mapping\ClassMetadata;
+use Symfony\Component\Validator\Mapping\Loader\AttributeLoader;
+
+class WordCountTest extends TestCase
+{
+    public function testLocaleIsSet()
+    {
+        $encoding = new WordCount(min: 1, locale: 'en');
```

Comment:
Why do you store the constraint in a variable named "encoding"?

Reply:
That's a mistake from mine, that's updated now